{"uuid": "4853fdbc-703d-4a97-b208-b65b27b1ff14", "children": ["43a97fc3-130a-4f46-8f87-5bbec12ffade"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114\nCurrent browser version is 116.0.5845.141 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00E7A813+48355]\n\t(No symbol) [0x00E0C4B1]\n\t(No symbol) [0x00D15358]\n\t(No symbol) [0x00D361AC]\n\t(No symbol) [0x00D31EF3]\n\t(No symbol) [0x00D30579]\n\t(No symbol) [0x00D60C55]\n\t(No symbol) [0x00D6093C]\n\t(No symbol) [0x00D5A536]\n\t(No symbol) [0x00D382DC]\n\t(No symbol) [0x00D393DD]\n\tGetHandleVerifier [0x010DAABD+2539405]\n\tGetHandleVerifier [0x0111A78F+2800735]\n\tGetHandleVerifier [0x0111456C+2775612]\n\tGetHandleVerifier [0x00F051E0+616112]\n\t(No symbol) [0x00E15F8C]\n\t(No symbol) [0x00E12328]\n\t(No symbol) [0x00E1240B]\n\t(No symbol) [0x00E04FF7]\n\tBaseThreadInitThunk [0x76FD00C9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77527B1E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77527AEE+238]\n\n", "trace": "  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\Ira\\Python\\stepik_auto_tests_course_final_block\\conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(options=options)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 81, in __init__\n    super().__init__(\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\irina\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1694430739194, "stop": 1694430743690}], "start": 1694430739194, "stop": 1694430743746}